backends/platform/sdl/sdl.cpp

```diff
@@ -752,3 +752,16 @@ int SDL_SetColorKey_replacement(SDL_Surface *surface, Uint32 flag, Uint32 key) {
 }
 #endif
 
+char *OSystem_SDL::convertEncoding(const char *to, const char *from, const char *string, size_t length) {
+#if SDL_VERSION_ATLEAST(2, 0, 0)
```

Comment:
Why? `SDL_iconv_string()` is available with SDL1 as well.

Reply:
When I first needed to convert encodings, I greped "SDL_iconv_string" in the whole ScummVM and I saw, that everytime it is used, it is wrapped inside #if SDL_VERSION_ATLEAST(2, 0, 0). So I assumed SDL2 is required for SDL_iconv_string. Now I looked at it again and it is most likely wrapped inside the #if #endif because of other functions used in those blocks of code. So I will remove that.